olm-catalog/serverless-operator/project.yaml

```diff
@@ -67,10 +67,5 @@ dependencies:
         eventing_kafka_broker: knative-v1.9
 upgrade_sequence:
     - csv: serverless-operator.v1.29.0
-      source: redhat-operators
-    - csv: serverless-operator.v1.29.1
-      source: redhat-operators
     - csv: serverless-operator.v1.30.0
-      source: serverless-operator
     - csv: serverless-operator.v1.31.0
-      source: serverless-operator
```

Comment:
we can remove this line now
https://github.com/openshift-knative/serverless-operator/blob/b2506aa104c5e78995354c5296bc44f6d771f7b2/hack/cmd/bumpso/bumpso.go#L85

Reply:
I think it's already removed in the current PR.